### PR TITLE
Update keras_CV guides_master.py

### PR DESCRIPTION
**Pull request by divyashreepathihalli**

### CODE REVIEW

File: scripts/guides_master.py [hunk 1]
```diff
@@ -23,6 +23,18 @@
             "path": "custom_image_augmentations",
             "title": "Custom Image Augmentations with BaseImageAugmentationLayer",
         },
+        {
+            "path": "semantic_segmentation_deeplab_v3_plus",
+            "title": "Semantic Segmentation with KerasCV",
+        },
+        {
+            "path": "coco_metrics",
```
Comment (ianstenbit) [inline scripts/guides_master.py:31]: I'm not sure if we want to publish these other 2 right now -- I'm not certain of the state they are in, and at least for the COCO one we already discuss this in the OD guide.

**Review by ianstenbit — APPROVED** [review]
LGTM for adding the segmentation one, but I think we can drop the other 2.

File: scripts/guides_master.py (hunk 1 above)
Comment (divyashreepathihalli) [reply scripts/guides_master.py:31]: got it! I'll remove them

**Comment by divyashreepathihalli** [comment]
> LGTM for adding the segmentation one, but I think we can drop the other 2.

done!

**Review by fchollet — APPROVED** [review]

**Pull request merged**